src/domain/Collection/BillList.js

```diff
@@ -69,15 +72,24 @@ const BillList = ({ fmk, isLoading }) => {
   const AutoFill = () => {
     let billsListCopy = createBillsCopy();
 
-    billsListCopy.forEach((row) => {
-      row.paidTotal = row.outstandingCost + row.outstandingTax;
-      row.paidCost = row.outstandingCost;
-      row.paidTax = row.outstandingTax;
-      row.method = 2183; // auto setting paid method to "Cheque"
-    });
-
+    if (!isToggled) {
```

Comment:
It should be ok but in terms of being intuitive, it is better not to have a negative comparison because you need to think one more time. So just make like
```
if (isReset) {
  // do for Reset button case
} else {
  // do for Autofill case
}
```